docs/build.md

```diff
@@ -71,6 +126,27 @@ After you do `yarn ios` and `yarn android` once, you can later just run `yarn we
   - For instance, the locally-hosted dev-wallet will need `adb reverse tcp:3001 tcp:3001`
 - For some reason, the typescript compiler chokes on platform-specific files (e.g. `foo.native.ts`) but only when compiling for Web thus far. Therefore we always have one version of the file that doesn't use a platform specifier, and that should be the Web version. ([More info](https://stackoverflow.com/questions/44001050/platform-specific-import-component-in-react-native-with-typescript).)
 
+### When in doubt, throw it out 🧨
+
+Sometimes the most effective fix for stubborn React Native/Expo issues is the nuclear option. If you've tried everything else and are still encountering mysterious errors:
```

Comment:
Very common "WTF, why isn't this working?" _quick-fixes-before-you-start-debugging-and-installing-stuff-you-do-not-need_ for RN projects in general.